resources/js/Components/Form/EntryCreateForm.tsx

```diff
@@ -0,0 +1,413 @@
+import {useForm} from '@inertiajs/inertia-react'
+import AdapterDateFns from '@mui/lab/AdapterDateFns'
+import DatePicker from '@mui/lab/DatePicker'
+import LocalizationProvider from '@mui/lab/LocalizationProvider'
+import {Button, Grid, MenuItem, TextField, Typography} from '@mui/material'
+import {ErrorText, Link, StepperForm} from 'Components'
+import React, {useState} from 'react'
+import styled from 'styled-components'
+import {adventureType} from 'Types/adventure-data'
+import {CharacterData} from 'Types/character-data'
+import {EntriesData} from 'Types/entries-data'
+import {ItemData} from 'Types/item-data'
+import {RatingCategoryType} from 'Types/rating-data'
+import route from 'ziggy-js'
+
+import ItemForm from './ItemForm'
+import RatingForm from './RatingForm'
+
+type EntryCreateFormPropType = {
+    type: 'Edit' | 'Create'
+    onCloseDrawer?: () => void
+    editData?: EntriesData
+    editId?: number
+    character: CharacterData
+    adventures: adventureType[]
+}
+
+type EntryFormDataType = {
+    adventure_id: number
+    location: string
+    length: number
+    levels: number
+    gp: number
+    date_played: string | null
+    dungeon_master: string | null
+    notes: string
+    items: ItemData[]
+    rating_data: RatingCategoryType
+    type: 'game'
+    character_id: number
+}
+
+const StyledDateGrid = styled(Grid)`
+    margin-top: 16px;
+    margin-bottom: 23px;
```

Comment:
23 is such a weird number lol

Reply:
I knooooow but it's really the one that makes the date field line up ;-;